src/lib/commits.js

```diff
@@ -0,0 +1,23 @@
+import { exec } from 'child_process'
+
+import { efh } from './error'
```

Comment:
FYI: In my recent Babel project I got back to use the regular `require` again, since module loading was recently kicked out of the spec (https://github.com/lukehoban/es6features/issues/75#issuecomment-97763850) and I wanted to keep the code as standard compatible as possible.